### PR TITLE
Emags emag emags

### PR DESCRIPTION
**Pull request by ghost**
Emags emag emags

:cl:
rscadd: Emags emag emags
/:cl:

emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag emag

### CODE REVIEW

**Comment by ghost** [comment]
God fucking damnit that shit with my master didn't work.  HOW DID IT GET WORSE WHAT

**Comment by Amelia0010** [comment]
wew

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -71,6 +71,25 @@
 	if(!proximity && prox_check)
 		return
 	A.emag_act(user)
+	
+//Don't let Nich touch my baby
```
Comment (Amelia0010) [inline code/game/objects/items/weapons/cards_ids.dm:75]: 🤔

**Review by SpaceJohn413 — APPROVED** [review]
yes

this is excellent memery

File: code/game/objects/items/weapons/cards_ids.dm [hunk 2]
```diff
@@ -71,6 +71,25 @@
...
+			if(user.l_hand != src)
+				otherEmag = user.l_hand
+	if(!otherEmag)
+		if(user.r_hand)
+			if(istype(user.r_hand, /obj/item/weapon/card/emag))
+				if(user.r_hand != src)
+					otherEmag = user.r_hand
+	if(!otherEmag)
+		return
+	to_chat(user, "<span class='notice'>The cyptographic sequencers attempt to override each other inevitably destroying themselves.</span>")
```
Comment (ghost) [inline code/game/objects/items/weapons/cards_ids.dm]: My three favorite things are eating my family and not using commas

in other words, use a fucking comma after 'other'. And inevitably is the wrong word to use here. Just drop it entirely.

File: code/game/objects/items/weapons/cards_ids.dm (hunk 1 above)
Comment (Notamaniac) [reply code/game/objects/items/weapons/cards_ids.dm:75]: Touch it. Touch it as hard as you can. Do it.

**Comment by Alek2ander** [comment]
Sound of sparks but no actual sparks.
:-1: :heavy_multiplication_x:  :100:

**Comment by Alexandria404** [comment]
The feature we always lacked

**Pull request merged**